### ISSUE TITLE
Refine architecture for deferred proofing via SIS & Lighthouse

### ISSUE THREAD

**Issue by alevindecanini** [department-of-veterans-affairs/va.gov-team#57182]

**Comment by alevindecanini** [comment]
@sdickson-gt can you please help fill in the description, tasks, and acceptance criteria for this ticket?

**Comment by alevindecanini** [comment]
@sdickson-gt when does this need to be completed relative to other deferred proofing tasks?

**Comment by alevindecanini** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Logingov-adoption-Team-632280b31e745000136a61fc/issues/department-of-veterans-affairs/va.gov-team/57182?planning-poker) with Zenhub @sdickson-gt

**Comment by sdickson-gt** [comment]
We could go live with deferred proofing on just USiP.  These are needed for a more comprehensive solution covering SiS and Lighthouse entry points.  This is dependent unblocking risks for deferred proofing and IAM implementation of binding accounts.
